### PR TITLE
Adds fuel rods to cargo console

### PR DESCRIPTION
**Pull request by TheFakeElon**
## About The Pull Request

You can now purchase uranium fuel rods for $4000

## Why It's Good For The Game

Just in case your original rods somehow go missing. also will be useful for a future PR...

## Changelog
:cl:
add: Fuel rods can now be bought from cargo
/:cl:

### CODE REVIEW

**Comment by thelaughingbomb** [comment]
Reactor construction kit when?

**Comment by TheFakeElon** [comment]
@thelaughingbomb hmm, I could make a beacon I guess. The way the reactor is coded doesn't allow for typical construction

**Comment by TheFakeElon** [comment]
I would love to see someone turn cargo into an enrichment centre though.. plutonium prices will be coming soon

**Comment by thelaughingbomb** [comment]
that sounds epic, you would need to make sure the 3 inputs are included in the beacon so people know which input is what

**Comment by Moccha-Bee** [comment]
why did travis shit itself so hard

**Review by MCterra10 — APPROVED** [review]

**Review by Moccha-Bee — APPROVED** [review]

File: austation/code/modules/cargo/packs/packs.dm [hunk 1]
```diff
@@ -9,3 +9,16 @@
...
+
+//Engineering
+
+/datum/supply_pack/engine/fuel_rod
+	name = "Uranium Fuel Rod crate"
+	desc = "Two additional fuel rods for use in a reactor, requires CE access to open. Caution: Radioactive"
+	cost = 4000
+	access = ACCESS_CE
+	contains = list(obj/item/twohanded/required/fuel_rod,
+					obj/item/twohanded/required/fuel_rod)
```
[the remark this answers is not in the document]
Comment (TheFakeElon) [reply austation/code/modules/cargo/packs/packs.dm]: It is

[the remark this answers is not in the document]
Comment (TheFakeElon) [reply austation/code/modules/cargo/packs/packs.dm]: you're viewing an outdated file

[the remark this answers is not in the document]
Comment (Moccha-Bee) [reply austation/code/modules/cargo/packs/packs.dm]: Oh, i just approved the changes, yeah i saw you fixed it, great pr by the way 👍🏻

**Pull request merged**